tests/js/spec/views/performance/landing/widgets/widgetContainer.spec.jsx

```diff
@@ -1146,90 +1146,87 @@ describe('Performance > Widgets > WidgetContainer', function () {
         })
       );
     });
-  });
-
-  it('TPM - Single Area Widget - metrics based', async function () {
```

Comment:
Did we remove the TPM test or am I reading the diff wrong?

Reply:
it was moved to be inside  `describe('Single Area Widget - metrics based', function () {`